### PR TITLE
docs: adding new readme files

### PR DESCRIPTION
**Pull request by himanshu010**
fixes: #1297 
**What kind of change does this PR introduce?**

docs related change
**Did you add tests for your changes?**
N/A
**If relevant, did you update the documentation?**
yes
**Summary**

**Does this PR introduce a breaking change?**

**Other information**

### CODE REVIEW

**Comment by himanshu010** [comment]
@rishabh3112 can you help me with the readme files of smoketest and docs

File: scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# scripts
+
+This directory contains javascripts file to build and prepare packages.
+
+## buildPackages.js
+
+```javascript
+'use strict';
+
+const fs = require('fs');
```
Comment (anshumanv) [inline scripts/README.md]: Do we really need the entire code in README?

Comment (snitin315) [reply scripts/README.md]: This is just 1 line readme otherwise.

Comment (himanshu010) [reply scripts/README.md]: I have seen some readme files which contains the entire code of important scripts files.
Screenshot (16)
https://github.com/webpack/webpack/tree/master/examples/aggressive-merging

therefore I inserted the whole code in the readme.

Comment (anshumanv) [reply scripts/README.md]: Can do that for important files like config as your example, but not sure if everyone needs to read the buildPackages (an example) code in a README, if they want they can always refer the actual file fwiw.

Comment (himanshu010) [reply scripts/README.md]: np i will remove these

Comment (himanshu010) [reply scripts/README.md]: Screenshot (17)
@anshumanv however these are also some of the files which are displayed and i think that they are not as important as the config file

Comment (anshumanv) [reply scripts/README.md]: We have a separate documentation site altogether with additional docs, for devs who want to actually get started with the codebase, what we have right now should suffice IMO. Anyways, that's just my say, would wait for thoughts from others. 😄

Comment (snitin315) [reply scripts/README.md]: I agree with @anshumanv

File: scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
+# scripts
```
Comment (rishabh3112) [inline scripts/README.md]: Having whole code don't make sense here. Any one can look up to the file and read the code.

Comment (rishabh3112) [reply scripts/README.md]: Please add short descriptions about the folders only, not the files as they are bound to change in logic in every PR.

File: .github/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+### Pull Request
+
+    -   [PULL_REQUEST_TEMPLATE](https://github.com/webpack/webpack-cli/tree/next/.github/PULL_REQUEST_TEMPLATE) : For new pull request
+
+## Others
+
+    -   [Workflows](https://github.com/webpack/webpack-cli/tree/next/.github/workflows)
+    -   [CONTRIBUTING.md](https://github.com/webpack/webpack-cli/tree/next/.github/CONTRIBUTING.md) : Contains informations regarding project setup and contribution
+    -   [FUNDING.yml](FUNDING.yml) : Contains information regarding funds collected by webpack organisation
```
Comment (evilebottnawi) [inline .github/README.md:26]: No need

File: test/README.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# test
+
+This directory contains all the neccesary tests to check to check that whether the project runs perfectly or is there some error.
+
+## Testing is essential because:
+
+-   To point out the defects and errors that were made during the development phases.
+-   It’s important to ensure that the application should not result into any failures
```
Comment (evilebottnawi) [inline test/README.md:8]: No need

**Pull request closed without merging**